src/main/scala/org/broadinstitute/hail/driver/ExportPlink.scala

```diff
@@ -25,6 +58,85 @@ object ExportPlink extends Command {
 
   def requiresVDS = true
 
+  def getFamRows(vds: VariantDataset, options: Options) : IndexedSeq[String] = {
```

Comment:
Should we also enforce that all these things result in specific types?  For sex in particular, there are a million ways to represent it -- isFemale (boolean), numeric(int), "male"/"female"...

Asking that the user explicitly figure out how to make it an int (which is what plink expects right?) might be the right thing to do